etc/adminhtml/menu.xml

```diff
@@ -54,5 +54,14 @@
             action="algolia_algoliasearch/reindex/index"
             resource="Algolia_AlgoliaSearch::manage"
         />
+        <add
+            id="Algolia_AlgoliaSearch::analytics"
+            title="Analytics Overview"
+            module="Algolia_AlgoliaSearch"
+            sortOrder="110"
+            parent="Algolia_AlgoliaSearch::manage"
+            action="algolia_algoliasearch/analytics/index"
+            resource="Algolia_AlgoliaSearch::manage"
+        />
     </menu>
 </config>
```

Comment:
Could you add "empty" line to each file ending like that? :) It's not super useful, but looks better :)))